sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/ReadBlobArgumentBinding.cs

```diff
@@ -39,5 +41,85 @@ public static async Task<Stream> TryBindStreamAsync(BlobBaseClient blob, Cancell
 
             return rawStream;
         }
+
+        public static async Task<Stream> TryBindStreamAsync(BlobBaseClient blob, string eTag, CancellationToken cancellationToken)
```

Comment:
This overload looks almost like the other one. I'd rather add optional `eTag` to other overload and create conditions if it's provided.